packages/flutter_test/lib/src/test_pointer.dart

```diff
@@ -437,3 +437,20 @@ class TestGesture {
     });
   }
 }
+
+/// A pack of input PointerEvent queue.
+///
+/// [timeStamp] is used to indicate the time when the pack is received.
+///
+/// This is a simulation of how the framework is receiving input events from
+/// the engine. See [GestureBinding] and [PointerDataPacket].
+class PointerEventPack {
```

Comment:
Is this the `PointerEvent` version of the `PointerDataPacket` in the sense that each `PointerData` in the `PointerDataPacket` is converted to a `PointerEvent`, and put into this packet?

If so, maybe `PointerEventPacket` is a better name to signal the relationship between this and `PointerDataPacket`.

Reply:
I now don't feel very comfortable with this renaming. `PointerDataPacket` is just a list of `PointerData` without timestamp https://api.flutter.dev/flutter/dart-ui/PointerDataPacket/PointerDataPacket.html but this class has a time stamp. I'm thinking if `PointerEventRecord` is a better name, and so is `PointerDataRecord` if we are going to change the API from accepting `PointerEvent` to `PointerData`.